plugins/org.yakindu.sct.generator.cpp/src/org/yakindu/sct/generator/cpp/StatemachineHeader.xtend

```diff
@@ -234,6 +256,10 @@ class StatemachineHeader extends org.yakindu.sct.generator.c.files.StatemachineH
 		return ''''''
 	}
 
+	def createInlineTR_Destructor(ExecutionFlow it) {
+		'''inline «module»::«tracingModule»::~«tracingModule»() {}'''
```

Comment:
You could just provide the empty implementation where you declared the destructor above.